secant/UIComponents/Buttons/NeumorphicDesignModifier.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  NeumorphicDesignModifier.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 02/21/22.
+//
+
+import SwiftUI
+
+/// Neumorphic design is charasterictical with two shadows (light & dark) around the view
+/// Appereance in our case is influenced by two parameters:
+/// - Parameters:
+///   - colorScheme: The light is using full neumorphic design while dark is limited to soft shadow only
+///   - isPressed: When the button is pressed, there are different behaviours for light vs. dark colorScheme
+/// This design is mostly used for CircularFrame, not designed for a button (see NeumorphicButtonDesign)
+struct NeumorphicDesign: ViewModifier {
```

Comment:
What would you think about reducing this naming to Neumorphic and NeumorphicButton?  Does the Design suffix add value?

Reply:
updated